src/ProjectTemplates/test/EmptyBlazorServerTemplateTest.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Testing;
+using Templates.Test.Helpers;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace Templates.Test;
+
+public class EmptyBlazorServerTemplateTest : BlazorTemplateTest
```

Comment:
This isn't running in the CI as there was some refactoring of the project template test classes recently. I believe you'll have to move this over into [here](https://github.com/dotnet/aspnetcore/tree/db608a5f8db4de960af55961c41f3e22a776d7cd/src/ProjectTemplates/test/Templates.Blazor.Tests)

https://github.com/dotnet/aspnetcore/tree/db608a5f8db4de960af55961c41f3e22a776d7cd/src/ProjectTemplates/test